### PR TITLE
Use lambda expressions

### PR DESCRIPTION
**Pull request by SoSeDiK**
Nicer code & reduced file size.

### CODE REVIEW

File: src/me/mrCookieSlime/Slimefun/SlimefunGuide.java [hunk 1]
```diff
@@ -562,7 +551,6 @@ else if (!(category instanceof LockedCategory)) {
 						if (!(category instanceof SeasonCategory)) {
 							menu.addItem(index, category.getItem());
 							menu.addMenuClickHandler(index, new MenuClickHandler() {
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/Slimefun/SlimefunGuide.java]: I suspect the file size could be reduced even further by replacing all MenuClickHandlers with
(p, slot, item, action) -> false

Comment (SoSeDiK) [reply src/me/mrCookieSlime/Slimefun/SlimefunGuide.java]: Alright, I'll do this.
Also, I can make a separate PR later using diamond operators everywhere if you don't mind.

**Comment by SoSeDiK** [comment]
Oh!
I'm really sorry to everyone who still maintain their custom forks with updates.
I respect you all.

**Comment by Trigary** [comment]
I wonder... The changes in this PR seem awfully similar. Oh, I remember now! #604

**Comment by SoSeDiK** [comment]
Yeah 👀
Though I've tried my best by then to reduce the *mess* by removing some repeating code blocks (SlimefunSetup even got its own PR #875; and the Altar was rewritten somewhere else I believe).

**Pull request merged**